sdk/python/tests/test_offline_online_store_consistency.py

```diff
@@ -133,10 +135,13 @@ def prep_local_fs_and_fv() -> Iterator[Tuple[FeatureStore, FeatureView]]:
             join_key="driver_id",
             value_type=ValueType.INT32,
         )
+        project = f"test_local_correctness_{str(uuid.uuid4()).replace('-', '')}"
+        print(f"Using project: {project}")
```

Comment:
Should this print be here?

Reply:
Since we generate the project dynamicaly and this is for integration tests, I think it's actually useful for debugging. I can remove it if you feel strongly about removing it

Reply:
I dont feel strongly.

Reply:
In that case I'm going to leave it in here